src/NECompletion-Morphic/CompletionEngine.class.st

```diff
@@ -10,8 +10,9 @@ The completion occurs in specific character position. The editor is responsible
 "
 Class {
 	#name : #CompletionEngine,
-	#superclass : #Object,
+	#superclass : #AbstractCompletionEngine,
```

Comment:
- Now CompletionEngine is part of Morphic (It should maybe be renamed MorphicCompletionEngine? RubricCompletionEngine?)
- An abstract superclass allows the system to discover all available completion engines and to document the API.

Yet, I don't think this is right in the long term if we have other editors than rubric, like a Gtk based editor. I leave this for a future iteration.